.infrastructure/11_dns.tf

```diff
@@ -0,0 +1,5 @@
+module "dns_zone" {
+  source  = "terraform-aws-modules/route53/aws//modules/zones"
```

Comment:
you should point to a precise commit
take a look here (https://github.com/pagopa/developer-portal/blob/dc5ece2b589f82bdcac1b625ad78b4290994bc6b/.infrastructure/55_cognito.tf#L16)

Reply:
ok, with new commit: source = "git::github.com/terraform-aws-modules/terraform-aws-route53.git?ref=fb53f9541723161ff39374add047dd8e60441e10" # v2.11.0